package.json

```diff
@@ -81,7 +81,7 @@
   },
   "packageManager": "yarn@3.2.1",
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.0.0"
```

Comment:
Thoughts on moving this to a separate PR? It would be easier to keep track of that way

Reply:
Yes, I think it would be better to separate the scopes into two PRs

I created [236](https://github.com/MetaMask/KeyringController/pull/236), that we could merge before this PR